app/views/hyrax/base/_workflow_actions.html.erb

```diff
@@ -12,7 +12,7 @@
         <% presenter.workflow.actions.each do |key, label| %>
           <div class="radio">
             <label>
-              <input type="radio" name="workflow_action[name]" id="workflow_action_name_<%= key %>" value="<%= key %>">
+              <%= radio_button_tag 'workflow_action[name]', key, :checked => 'checked' %>
```

Comment:
This will make every radio button default to checked.  Shouldn't it only be one of them?

Reply:
Yeah - i was having trouble with any conditional logic not producing the same result. But I think I've fixed that. The 3rd parameter here needs to be just true/false, not an option for `checked`. It should now only be checked on `comment_only`